### DISCUSSION TITLE
Hello, how do you give players claim blocks? (not over time / accrued)

### DISCUSSION THREAD

**Discussion by FauxOW** [GriefPrevention/GriefPrevention#804] [Q&A] [answered]
The answer to your question may exist in the wiki, go check that out first! 

Wiki: https://github.com/TechFortress/GriefPrevention/wiki

Otherwise, if you have a spigot account, you may have better luck posting in the discussion thread so people who follow the thread may see your question. https://www.spigotmc.org/threads/griefprevention.35615/page-311

Still no luck? Really? Then go ahead and ask here.

**Comment by RoboMWM** [comment]
Well, did you check the wiki?

**Comment by Jikoo** [comment, marked as answer]
You can configure the amount of blocks that players get when starting in your configuration. If you want to give them more later, there are [admin commands](https://github.com/TechFortress/GriefPrevention/wiki/Commands#admin-commands) for adding bonus blocks or changing base blocks for players.

**Reply by Meyexe** [reply]
but what if i dont want to give bonus blocks and give Accrued blocks instead manually

**Reply by Jikoo** [reply]
You'd have to do some math and use `/SetAccruedClaimBlocks`.
